tests/test_dfa.py

```diff
@@ -2504,3 +2504,20 @@ def test_empty_language(self) -> None:
 
         dfa = DFA.empty_language({"0", "1", "a", "b"})
         self.assertTrue(dfa.isempty())
+
+    def test_reset_word_cache(self) -> None:
+        max_len = 4
+        dfa = DFA.of_length(set("01"), min_length=0, max_length=max_len)
```

Comment:
@eliotwrobson This is totally a trivial nitpick, but you can use a set literal here instead of the `set()` function (i.e. `{"0", "1"}`). This is how it's written for the other `DFA.of_length` calls in the codebase.

Reply:
Done here https://github.com/caleb531/automata/pull/152/commits/8ba372fcfdbe58e87fd5128113f13e5ca889903f